transport_plugins/bled112/RELEASE.md

```diff
@@ -2,6 +2,10 @@
 
 All major changes in each released version of the bled112 transport plugin are listed here.
 
+## 1.8.0
```

Comment:
Let's  not release this just yet.  This is a core code path that affects all coretools installations including those on nodes and gateways.  If you want to do a prerelease to make this easier to install and test, go for that otherwise, since no one else has v2 advertising packets, you can test locally with a `pip install -e <path to folder>`